conductr_cli/conduct_main.py

```diff
@@ -283,7 +283,7 @@ def build_parser(dcos_mode):
 
 def get_cli_parameters(args):
     parameters = ['']
-    if not args.dcos_mode:
+    if not vars(args).get('dcos_mode'):
```

Comment:
`dcos_mode` is now always set.

I'm keeping `vars(args).get('dcos_mode')` since it's a safe way to access value within `args`, and also it's consistent with how other fields being accessed.